datasets/multi_woz_v22/multi_woz_v22.py

```diff
@@ -0,0 +1,282 @@
+                            ),
+                            "dialogue_acts": datasets.Features(
+                                {
+                                    "dialog_act": datasets.Sequence(
+                                        datasets.Features(
+                                            {
+                                                "act_type": datasets.Value("string"),
+                                                "act_slots": datasets.Sequence(
+                                                    datasets.Features(
+                                                        {
+                                                            "slot_name": datasets.Value("string"),
+                                                            "slot_value": datasets.Value("string"),
+                                                        }
+                                                    ),
+                                                ),
+                                            }
+                                        )
+                                    ),
+                                    "span_info": datasets.Sequence(
+                                        datasets.Features(
+                                            {
+                                                "act_type": datasets.Value("string"),
+                                                "act_slot_name": datasets.Value("string"),
+                                                "act_slot_value": datasets.Value("string"),
+                                                "span_start": datasets.Value("int32"),
+                                                "span_end": datasets.Value("int32"),
+                                            }
+                                        )
+                                    ),
+                                }
+                            ),
+                        }
+                    )
+                ),
+            }
+        )
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,  # Here we define them above because they are different between the two configurations
+            supervised_keys=None,
+            homepage="https://github.com/budzianowski/multiwoz/tree/master/data/MultiWOZ_2.2",
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        data_files = dl_manager.download_and_extract(_URLs)
+        self.stored_dialogue_acts = json.load(open(data_files["dialogue_acts"]))
+        return [
+            datasets.SplitGenerator(
+                name=spl,
+                # These kwargs will be passed to _generate_examples
+                gen_kwargs={
+                    "filepaths": data_files,
+                    "split": spl,
+                },
+            )
+            for spl in ["train", "dev", "test"]
+        ]
```

Comment:
can you use `datasets.Split.TRAIN`, `datasets.Split.VALIDATION` and `datasets.Split.TEST` if possible ?